.github/workflows/test-keytools.yml

```diff
@@ -144,3 +144,9 @@ jobs:
       - name: Generate final signed binary
         run: |
           ./tools/keytools/sign --rsa2048 --sha256 --manual-sign test-app/image.elf public-key.der 1 test-app/image_v1.sig
+
+
+    # TODO: Test WOLFBOOT_UNIVERSAL_KEYSTORE
```

Comment:
Do you want to add these tests now or future PR?

Reply:
good idea, I'll add them now